gratipay/email.py

```diff
@@ -86,9 +86,10 @@ def put(self, to, template, _user_initiated=True, **context):
                     raise Throttled()
 
 
-    def flush(self):
+    def flush(self, _flush_one=None):
         """Load messages queued for sending, and send them.
         """
+        flush_one = _flush_one or self._flush_one
```

Comment:
Hmm, why is this needed? We've got a console mailer hooked up in the test env, no?